src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,111 @@
 package core.basesyntax;
```

Comment:
Super work. But I have see some controversial issues.

Reply:
Thank you and the advice was sound)